book/book.tex

```diff
@@ -47,6 +47,7 @@ \chapter{Non wheat sourdough}%
 \label{chapter:non-wheat-sourdough}
 \input{non-wheat-sourdough/non-wheat-sourdough}
 
+\input{mix-ins/mix-ins.tex}
```

Comment:
```suggestion
% \input{mix-ins/mix-ins.tex}
```
I suggest we comment this for now until it is finished.

Reply:
it was in... I was not going to remove as you saw it fit.  I am more of a branch rather than comment kind of guys, but maybe here commenting it out is the right thing to do.